src/components/Tab.tsx

```diff
@@ -0,0 +1,41 @@
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles'
+import Tab from '@material-ui/core/Tab'
+
+interface StyledTabProps {
+  label: string
+}
+
+export default withStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      textTransform: 'none',
+      minWidth: 72,
+      backgroundColor: 'transparent',
+      fontWeight: theme.typography.fontWeightRegular,
+      marginRight: theme.spacing(4),
+      fontFamily: [
+        '-apple-system',
+        'BlinkMacSystemFont',
+        '"Segoe UI"',
+        'Roboto',
+        '"Helvetica Neue"',
+        'Arial',
+        'sans-serif',
+        '"Apple Color Emoji"',
+        '"Segoe UI Emoji"',
+        '"Segoe UI Symbol"',
+      ].join(','),
```

Comment:
I see you call it root, but does it effect everything, or is this going to be reused anywhere else? In that case I'd suggest extracting it to the theme or some other place as a constant.